src/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
```

Comment:
I think the setup.py file needs to be one dir level up from the package contents. So then the setup.py would be at the repo root and the `src` dir becomes the top level of the package and we can import using `import src`.

If we don't want to have it in the repo root dir (eg so as not to conflict with DE stuff), we can create a subdir under `src` to contain the modules.

Reply:
My current understanding is: The name of the package used in import statements is determined by the name of the folder that the setup.py file is in. 

So, if we put the setup.py file in the repo root, then the name of the package used in import statements would be `search-terms-sanitization`. Is there another way to change this?

Reply:
Hm, I think what is available in import statements is what is passed to the `packages` arg in setup.py.

I tested this out in a fresh env: put this setup.py in the root dir and run `pip install .`
```python
# setup.py
from setuptools import setup

setup(
    name="my_package",
    version="0.1.0",
    packages=["src"],
)
```
Then `pip list` shows `my-package` (underscore changed to dash), and in python I can do `import src` but not `import my_package`.

Reply:
ah okay -- thanks! I have made those changes